htdocs/update.bml

```diff
@@ -469,6 +469,22 @@ _c?>
 
                     my @after_entry_post_extra_options = LJ::Hooks::run_hooks('after_entry_post_extra_options', user => $ju, itemlink => $itemlink);
                     my $after_entry_post_extra_options = join('', map {$_->[0]} @after_entry_post_extra_options) || '';
+                    
+                    if ($req{"security"} eq "private") {
+                        $$body .=" p?><?p $ML{'.extradata.sec.private'}";
+                    } elsif ($req{"security"} eq "usemask") {
+                        if ($req{"allowmask"} == 0) { # custom security with no group -- essentially private
+                            $$body .=" p?><?p $ML{'.extradata.sec.private'}";
+                        } elsif ($req{"allowmask"} > 1) { # custom group
+                            $$body .=" p?><?p $ML{'.extradata.sec.custom'}";
+                        } else { # friends only
+                            $$body .=" p?><?p $ML{'.extradata.sec.access'}";
+                        }
+                    } else {
+                        $$body .=" p?><?p $ML{'.extradata.sec.public'}";
+                    }
+                
+                    $$body .=" p?><?p " . BML::ml('.extradata.subject', { subject => $req{"subject"} });
```

Comment:
Make sure to escape the subject here for safety, because this is user supplied (LJ::ehtml -- see how other portions of the page do it).